Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -362,6 +362,38 @@ div.bocList div.bocListTableBlock div.bocListNavigator a:not([href]) img
 }
 /*endregion*/
 
+/*region bocListValidationRow*/
+div.bocList div.bocListTableBlock tr.bocListValidationRow
+{
+  border-bottom: var(--boclist-tabledata-border);
+  border-top: var(--boclist-tabledata-border);
+  border-top-style: dashed;
+  color: red;
+}
+
+div.bocList table.bocListTable tr.bocListValidationRow.selected td
+{
+  background-color: var(--color-highlight);
+}
+
+div.bocList div.bocListTableBlock tr.bocListValidationRow td ul
+{
+  margin: calc(var(--remotion-themed-spacing) / 2) var(--remotion-themed-spacing);
+}
+
+div.bocList div.bocListTableBlock tr.bocListValidationRow td.bocListValidationRowStickyCell
+{
+  position: sticky;
+  left: 0;
+  white-space: nowrap;
+}
+
+div.bocList div.bocListTableBlock tr.bocListValidationRow td.bocListValidationRowStickyCell ul
+{
+  max-width: 1px;
```

Comment:
The current solution with the special table cell setup and the 1px column to get the sticky effect feels too complex for me. In addition, when we extend the solution to have a JS component that helps with sizing, we would need to rewrite a lot of it anyway.

I experimented a bit and came up with the following solution:
![image](https://github.com/re-motion/Framework/assets/5082673/88ffcf4d-a4c2-4511-86ba-d9463b3e898f)

 - (This CSS rule and the rules above were disabled for this demo)
 - The `td` element now spans the whole row with a `colspan` (this should be set to the exact number of columns instead of `99` in the end)
 - The `ul` element is wrapped in two `div` elements
   - The first provides overflow+sticky logic and should always at most as big as the visible space
     - JS would set the size of this div to the visible width, providing the correct sticky behavior
     - For now I just added a `max-width` to show the sticky behavior
   - The second div is responsible for the sizing of the text. By using `min-inline-size` the text is forced to take at least that much space, overflowing in tight scenarios like we want

I think this setup provides a clean solution that we can extend later on by setting the width using JS